mlir/include/mlir/Transforms/Passes.h

```diff
@@ -43,6 +43,7 @@ class GreedyRewriteConfig;
 #define GEN_PASS_DECL_SYMBOLDCE
 #define GEN_PASS_DECL_SYMBOLPRIVATIZE
 #define GEN_PASS_DECL_TOPOLOGICALSORT
+#define GEN_PASS_DECL_COMPOSITEFIXEDPOINTPASS
```

Comment:
Could we just replace all this list with `#define GEN_PASS_DECL`? Are there passes we don't want here?

Reply:
There are `ViewOpGraph` and `LocationSnapshot` passes which are not listed here but listed in separate header files, so if I add`GEN_PASS_DECL` here it fails with struct redefinition error.